content/resources/user/submit-data/sower.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "Gen3 - Data Contribution"
+date: 2018-09-12T20:53:20-05:00
+linktitle: /resources/user/submit-data
+layout: withtoc
+menuname: userMenu
+---
+{{% markdownwrapper %}}
+# Linking Data from external Data Clouds to Gen3 Data Commons
+***
+
+It is possible to link data on Gen3 that is stored on other cloud services (Amazon Web Services AWS, Google Cloud Storage GCS) by a process called DIIRM (*Data Ingestion, Integration, and Release Management*). In other words, if you have a bucket of files and want to link the data to Gen3 you can find below a step-by-step guide and a practical example. Before going forward, you need to make sure you have access to the external bucket.
+
+1. [Step-by-step guide](#1-step-by-step-guide)
+2. [Practical example with 1000 Genomes](#2-practical-example-with-1000-genomes)
+
+
+## 1. Step-by-step guide
+* * *
+- Create a manifest as a TSV file that contains all files that exist in the respective bucket. This manifest **has** to contain the following properties at the minimum: <ins>md5</ins> checksum, file <ins>size</ins> in bytes, and the full bucket <ins>urls</ins>. The column heads in the example manifest below should be taken as a reference as the indexing process is case- and word-sensitive.
+
+    ```
+    GUID	url	size	md5	acl
+     	s3://test/test.csv	745	613ccfbc68287db60c663519dab6a4ff	['*']
+    ```
+
+>__NOTE:__ If you want to authorize the access to the files and want a bucket to be tied to a project, you need to change `acl` and/or insert a column with `authz` as column head to add consent groups and index all the files with the project's `authz`.
```

Comment:
For consistency's sake, I would change "head" to "header".

Reply:
done